sagemaker_inference/files/serve

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+# This file implements the scoring service shell. You don't necessarily need to modify it for various
+# algorithms. It starts nginx and gunicorn with the correct configurations and then simply waits until
+# gunicorn exits.
+#
+# The flask server is specified to be the app object in wsgi.py
+#
+# We set the following parameters:
+#
+# Parameter                Environment Variable              Default Value
+# ---------                --------------------              -------------
+# number of workers        MODEL_SERVER_WORKERS              the number of CPU cores
+# timeout                  MODEL_SERVER_TIMEOUT              60 seconds
+
+import multiprocessing
```

Comment:
I think you can assume some things are happening for you, so i think this logic possibly exists in the platform and does not need to be replicated. as far as i understand you can simply define your api, which you have done in `task.py` and then name that file `serve` so that i can be called by sagemaker.